server/README.md

```diff
@@ -117,14 +110,10 @@ You should install these package to build Hatohol and required libraries.
 - libmysqlclient-dev
 - mysql-server
 - uuid-dev
-- qpidd
-- libqpidmessaging2-dev
-- libqpidtypes1-dev
-- libqpidcommon2-dev
 
 installing by following commands:
 
-    $ sudo apt-get install automake g++ libtool libsoup2.4-dev libjson-glib-dev libsqlite3-dev libmysqlclient-dev mysql-server sqlite3 uuid-dev qpidd libqpidmessaging2-dev libqpidtypes1-dev libqpidcommon2-dev
+    $ sudo apt-get install automake g++ libtool libsoup2.4-dev libjson-glib-dev libsqlite3-dev libmysqlclient-dev mysql-server sqlite3 uuid-dev
```

Comment:
> Should we remove Hatohol project repository descriptions?

I think so too.
We should install json-glib and devel package from centos base repository.
Please add  "json-glib" and "json-glib-devel" in this line.
And remove "For json-glib, there are two ways to install." section.